### DISCUSSION TITLE
openclash能上外网但唯独打不开youtube

### DISCUSSION THREAD

**Discussion by mark-chen1999** [vernesong/OpenClash#2747] [Q&A]
我想问一下我的openclash v0.45.59-beta在启动之后能上外网，但就是打不开youtube，在windows终端里nslookup www.youtube.com结果显示address：202.182.98.125，似乎并没有被正确解析？openclash的控制面板之中也只是显示这个ip，并没有显示出youtube的域名

我有做过其他的确认：
1. 配置文件没有问题，因为同样的配置文件在clash for windows中没有问题
2. openclash已打开dns劫持，DHCP中DNS转发确认是127.0.0.1#7874
3. openclash的各个核心均为最新版本
4. 软路由中并没有打开ipv6的设置，也没有其他插件处于开启状态

求解！

**Comment by eddieY1** [comment]
我也又这个问题，你解决了吗？求解

**Comment by mark-chen1999** [comment]
https://www.youtube.com/watch?v=7qiAZHc9n1Y

我是清除本地DNS缓存解决的。在cmd里运行ipconfig /flushdns

**Comment by bingoguo93** [comment]
我有时也这样，转换规则换了online full就可以匹配到走openclash，多用几天试试。

**Comment by ZwxwZ** [comment]
全局走代理就可以，我猜是ip不匹配规则
